api/cpp/cbindgen.rs

```diff
@@ -573,6 +573,19 @@ fn gen_backend_qt(
         .with_config(config)
         .with_crate(crate_dir)
         .with_include("slint_internal.h")
+        .with_after_include(
+            r"
+            namespace slint::cbindgen_private {
+                // HACK ALERT: This struct declaration is duplicated in internal/backend/qt/qt_widgets.rs - keep in sync.
+                struct SlintTypeErasedWidget
+                {
+                    virtual ~SlintTypeErasedWidget();
+                    virtual void *qwidget() = 0;
+                };
```

Comment:
```suggestion
                struct SlintTypeErasedWidget
                {
                    virtual ~SlintTypeErasedWidget() = 0 {}
                    SlintTypeErasedWidget(const SlintTypeErasedWidget&) = delete;
                    SlintTypeErasedWidget& operator=(const SlintTypeErasedWidget&) = delete;

                    virtual void *qwidget() const = 0;
```

deleted copy, and added const, because why not.
made destructor pure,  as well as inline so we are sure that we don't get link error.

Reply:
Forgot to mention that this should be kept in sync with the one in qt backend.

Reply:
I couldn't get the inline pure virtual destructor to compile :( So I kept it.

Similarly, I could make the qwidget() function const, but it needs to continue to return a void *, which means we'd have a const_cast on the impl. Not sure the const is worth it here, as we return `NonNull` anyway, which is `*mut`.

Reply:
ah yeah, don't make it const then.